CYBRHardeningCheck/Main.ps1

```diff
@@ -24,17 +25,25 @@ $global:InDebug = $PSBoundParameters.Debug.IsPresent
 $global:InVerbose = $PSBoundParameters.Verbose.IsPresent
 
 # Script Version
-$ScriptVersion = "3.1"
+$ScriptVersion = "3.1.1"
 
-# Set Log file path
-$global:LOG_FILE_PATH = "$ScriptLocation\Hardening_HealthCheck.log"
+# Set Date Time Pattern
+[string]$global:g_DateTimePattern = "$([System.Globalization.CultureInfo]::CurrentCulture.DateTimeFormat.ShortDatePattern) $([System.Globalization.CultureInfo]::CurrentCulture.DateTimeFormat.LongTimePattern)"
+
+# Set the script time stamp
+$reportDateTime = $(Get-Date -Format $g_DateTimePattern)
+# $reportDateTime = (Get-Date).toString("yyyy-MM-dd_HH_mm_ss")
+
+# Set output files paths
+# $global:LOG_FILE_PATH = "$ScriptLocation\Hardening_HealthCheck.log"
+# Time stamp added to the log name (emilg@segmentech.com)
+$global:LOG_FILE_PATH = "$ScriptLocation\Hardening_HealthCheck_$($reportDateTime.Replace("/","-").Replace(":","-").Replace(" ","_")).log"
```

Comment:
If the log file and the report needs to use the same file pattern with dates, worth having a parameter to hold that information once for all the places that are using it and concatenating it where it is needed no?